analyzers/src/SonarAnalyzer.CSharp/Helpers/CountComparisonSyntaxHelper.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+
+namespace SonarAnalyzer.Helpers
```

Comment:
We're moving extensions to `Extensions` directory and namespace to clean up the code base.